### PR TITLE
Update wget recipe

### PR DESCRIPTION
**Pull request by fingolfin**
Draft while PRs #2078 and #2079 are not merged.

### CODE REVIEW

**Comment by fingolfin** [comment]
Once GnuTLS_jll is in the registry, can rerun the freebsd job

File: W/wget/build_tarballs.jl [hunk 1]
```diff
@@ -7,32 +7,24 @@ version = v"1.20.3"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://ftp.gnu.org/gnu/wget/wget-1.20.3.tar.gz", "31cccfc6630528db1c8e3a06f6decf2a370060b982841cfab2b8677400a5092e")
+    ArchiveSource("https://ftp.gnu.org/gnu/wget/wget-$(version).tar.gz",
+                  "31cccfc6630528db1c8e3a06f6decf2a370060b982841cfab2b8677400a5092e")
```
Comment (giordano) [inline W/wget/build_tarballs.jl:11]:
```suggestion
                  "31cccfc6630528db1c8e3a06f6decf2a370060b982841cfab2b8677400a5092e"),
```

**Comment by giordano** [comment]
PkgServer seems sad today

**Comment by fingolfin** [comment]
T'is true. Need to buy it some chocolate to cheer it up

**Comment by fingolfin** [comment]
So this still pulls GnuTLS_jll v3.6.13+0 and not +1. :-(

**Pull request merged**